taxonomy/management/commands/refresh_job_skills.py

```diff
@@ -17,9 +17,11 @@
 
 class Command(BaseCommand):
     """
-        Example usage:
-        Create or refresh the existing data associated with jobs and job skills on the basis
-        of ranking_facet and nested_ranking_facet arguments.
+    Command to refresh the jobs associated with skills.
+
+    Example usage:
+        $ # Create or refresh the existing data associated with jobs and job skills on the basis
+        $ # of ranking_facet and nested_ranking_facet arguments.
```

Comment:
Please see comment about `#` character above.

Reply:
same as above, I added them to indicate that these are comments and not the command, bash comments start with #